src/styles.css

```diff
@@ -1,59 +1,94 @@
+@font-face {
+  font-family: "Final_Fantasy_VII";
+  src: url(./font/Final_Fantasy_VII.ttf) format("truetype");
+}
+
 * {
   padding: 0;
   margin: 0;
-  font-family: 'Comic Neue', cursive;
+ /* stylelint-disable */
+  font-family: "Final_Fantasy_VII";
+  /* stylelint-enable */
   list-style: none;
 }
 
+body {
+  background: url('./images/pngegg2.png') top right no-repeat;
+  background-size: 50%;
+  background-color: rgb(29, 27, 27);
+}
+
+/* .title,
+.score-title,
+.form-title {
+  font-family: "Final_Fantasy_VII";
+} */
+
 .title {
   margin: 2% 1%;
+  font-size: 4.5rem;
+  color: white;
+}
+
+.score-title,
+.form-title {
+  margin: 0 5%;
   font-size: 2.5rem;
+  color: white;
+}
+
+.score-title {
+  padding-top: 8%;
 }
 
-h2 {
-  margin: 0 4%;
-  font-size: 1.5rem;
+.form-title {
+  padding-top: 10%;
+  margin-bottom: 5%;
 }
 
 .subtitle-container {
   display: flex;
+  align-items: center;
 }
 
-.refresh-btn {
+.refresh-btn,
+.submit-btn {
   cursor: pointer;
-  padding-inline: 2%;
+  padding-inline: 5%;
+  font-size: 2rem;
+  padding-top: 6%;
+}
+
+.submit-btn {
+  padding-top: 12%;
+  margin-top: 10%;
 }
 
 .hard-list {
   box-sizing: border-box;
-  margin: 2% 4%;
-  border: 2px solid black;
-  list-style: none;
+  margin: 2% 2%;
+  border: 2px solid white;
+  border-radius: 4px;
+  font-size: 2rem;
+  padding: 3%;
+  background-image: linear-gradient(blue, rgb(28, 28, 109));
+  color: white;
 }
 
 .score-item > li {
   box-sizing: border-box;
-  margin: 5% 4%;
-  border: 2px solid black;
-  list-style: none;
-}
 
-/* .book-infocss {
-  display: flex;
-  width: 75%;
-  justify-content: space-between;
-  background-color: #fff;
-  align-self: center;
-  box-sizing: border-box;
-  margin: 2% 4%;
+  /* margin: 5% 4%; */
   border: 2px solid black;
   list-style: none;
-} */
 
-li:nth-child(odd) {
-  background-color: #e9d7fa;
+  /* padding-top: 6%; */
 }
 
```

Comment:
**I'm not able to inline in the particular file , Am sorry :pray:**
- [ ] You did a great job on this project but I noticed that we are having an issue with adding scores to the API. One possible reason for this is that the API endpoint URL might be incorrect. Please check that the API endpoint URL is correct and matches the URL provided by the service. also double-check that the unique ID is correct and that the endpoint is accessible.
Since we are not able to submit the scores, it could be better if you fix this :point_down: 

https://user-images.githubusercontent.com/88894525/235281006-f0d9e63d-6cf5-4425-9bde-95de970c1c55.mp4

_note: I can see that the problem is with your constructor function. The constructor function should not take any arguments in your case, as you are not passing any arguments when creating a new instance of Score. :wink: 
Also, the addScore method is not receiving the score data properly. You should pass an object with user and score properties instead of passing `scoreNumber` as a separate argument._

Reply:
Thank you Alfred! I changed the API URL and also rebuild the styles to check what happened. I tested the styles in the other branch, and now it is fixed :) I have corrected the issues in the current branch and now I'm able to add new user's scores 
![leaderboard2fixed](https://user-images.githubusercontent.com/120128067/235283159-b9ba924f-bd93-4d85-9d46-51f4fad1231f.jpg)